### PR TITLE
Sidebar layout padding

### PR DESCRIPTION
**Pull request by siggakogk**
# Layout spacing

Remove styling from rich text component
Make sidebarlayout handle padding for it's content
Move richtext into grid layout in newsitem and life events (where sidebarlayout is not being used)

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Formatting passes locally with my changes
- [ ] I have rebased against main before asking for a review

### CODE REVIEW

File: apps/web/components/RichText/RichText.tsx [hunk 1]
```diff
@@ -45,18 +40,7 @@ const renderComponent = (
...
-      <GridRow>
-        <GridColumn
-          offset={['0', '0', '0', '0', '1/9']}
-          span={['9/9', '9/9', '9/9', '9/9', '7/9']}
-        >
-          {children}
-        </GridColumn>
-      </GridRow>
-    )
+    children
```
Comment (barabrian) [inline apps/web/components/RichText/RichText.tsx:43]: what is this code doing?

Comment (barabrian) [reply apps/web/components/RichText/RichText.tsx:43]: maybe remove the if statement if this is no longer relevant

File: apps/web/screens/AboutSubPage/AboutSubPage.tsx [hunk 2]
```diff
@@ -77,38 +75,29 @@ export const AboutSubPage: Screen<AboutSubPageProps> = ({
...
-          </GridRow>
+          <Stack space={[3, 3, 2]}>
+            <Breadcrumbs>
+              <Link href="/">Ísland.is</Link>
+              <Link href="/stafraent-island">{parentPage.title}</Link>
+            </Breadcrumbs>
+            <Box display={['block', 'block', 'none']}>
+              <Navigation
+                baseId={'mobileNav'}
+                isMenuDialog={true}
```
Comment (barabrian) [inline apps/web/screens/AboutSubPage/AboutSubPage.tsx:86]:
```suggestion
                isMenuDialog
```

File: apps/web/screens/AboutPage/AboutPage.tsx [hunk 3]
```diff
@@ -186,14 +186,13 @@ const PageHeader: FC<PageHeaderProps> = ({
     <Background id={slice.id} theme={page.theme}>
       <GridContainer position="none">
         <ColorSchemeContext.Provider value={{ colorScheme: 'white' }}>
-          <Box marginBottom={[0, 0, 8, 15]}>
             <Header buttonColorScheme="negative" megaMenuData={megaMenuData} />
-          </Box>
         </ColorSchemeContext.Provider>
       </GridContainer>
       <SidebarLayout
         isSticky={false}
```
Comment (barabrian) [inline apps/web/screens/AboutPage/AboutPage.tsx:193]: is this not false by default?

**Review by barabrian — CHANGES_REQUESTED** [review]
minor comments

File: apps/web/screens/AboutPage/AboutPage.tsx (hunk 3 above)
Comment (siggakogk) [reply apps/web/screens/AboutPage/AboutPage.tsx:193]: it's true by default

**Review by barabrian — APPROVED** [review]

**Pull request merged**